mod/carrierwave/lib/carrier_wave/file_card_uploader.rb

```diff
@@ -175,6 +175,10 @@ def url_filename
       end
     end
 
+    # def temporary_identifier
+    #   db_content
+    # end
```

Comment:
There were several issues caused by these "temporary identifiers". It has to do with what gets written to the `mount_on` field (in our case, `#db_content`). This fixed certain problems but caused several others.